apps/condo/domains/notification/schema/NotifiableDevice.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Generated by `createschema notification.NotifiableDevice 'deviceId:Text; token?:Text; serviceType?:Select:firebase,apple,huawei; owner?:Relationship:User:SET_NULL; meta?:Json'`
+ */
+
+const { Text, Relationship, Select } = require('@keystonejs/fields')
+
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const { REQUIRED_NO_VALUE_ERROR } = require('@condo/domains/common/constants/errors')
+
+const access = require('@condo/domains/notification/access/NotifiableDevice')
+
+const { DEVICE_SERVICE_TYPE_KEYS } = require('../constants/constants')
+
+const NotifiableDevice = new GQLListSchema('NotifiableDevice', {
```

Comment:
just Device! We don't have non-notifiable devices ...

Reply:
fixed